docs/install/install_instructions.md

```diff
@@ -116,22 +118,207 @@ pip3 install neuron
 Note that Python2 wheels are provided for the 8.0.x release series exclusively. Also, we are not providing .rpm or .deb
 installers for recent releases.
 
-#### Windows
 
-On Windows, the only recommended way to install NEURON is using the binary installer. You can download alpha
-or recent releases from:
+---------------------- 
+
+## Windows
+
+Neuron Simulator detailed installation in Window.
 
-* [Alpha releases](https://neuron.yale.edu/ftp/neuron/versions/alpha/)
-* [Recent Releases](https://neuron.yale.edu/ftp/neuron/versions/)
+On Windows, the only recommended way to install NEURON is using the binary installer. You can download recent releases or any previous released version from:
+* [Recent Releases](https://github.com/neuronsimulator/nrn/releases/download/8.2.4/nrn-8.2.4.w64-mingw-py-37-38-39-310-311-setup.exe).
+* [List of Releases](https://github.com/neuronsimulator/nrn/releases/)
+
+---------------------- 
+
+#### Step 1: Download the NEURON for Windows Installer [here](https://github.com/neuronsimulator/nrn/releases/download/8.2.4/nrn-8.2.4.w64-mingw-py-37-38-39-310-311-setup.exe). 
+See the file name downloaded. 
 
 The naming convention for Windows installers is `nrn-<version-id>-mingw-py-38-39-310-311-setup.exe`.
 The `py-38-39-310-311` string in the installer name indicates that the given installer is compatible
-with Python versions 3.8, 3.9, 3.10 and 3.11. Once the installer is downloaded, you can install it
-by double clicking like any other Windows application. Note that you have to install python separately
-if python support is required. You can find detailed step-by-step instructions in
+with Python versions 3.8, 3.9, 3.10 and 3.11. 
+
+That means the last python version compatible is 3.11 
+
+---------------------- 
+
+#### Step 2: Install conda by [miniconda](https://docs.anaconda.com/free/miniconda/)  
+Follow the installation box steps. Select "Add Miniconda3 to my PATH environment variable" option, otherwise you will need do it manually after. 
+This picture: 
+ 
+![miniconda installation dialog box showing checked box](images/Install_conda_by_miniconda_1.png)
+ 
+Or this one: 
+
+![zoomed in view of the checked box in miniconda installation](images/Install_conda_by_miniconda_2.png)
+ 
+#### Step 3: Open a Windows PowerShell terminal (or CDM) 
+ 
+---------------------- 
+
+#### Step 4: Install Python via conda  
+
+`conda install python=3.#`
+where # (the last compatible version) 
+ 
+```bash
+conda install python=3.11 
+```
+We recommend installing Python via conda and not via Microsoft Store to avoid manually changing the PATH file. 
+ 
+---------------------- 
+
+#### Step 5: check python version 
+
+```bash
+python --version 
+```
+If the version is not compatible with neuron do: 
+
+`conda install python=X.XX`   ( See Step 4)
+
+(X.XX the last python version compatible with neuron. Yes, it may be a downgrade) 
```

Comment:
This could be greatly simplified with
```
winget install Python.Python.3.11
```
and the user will get an official Python, not the Conda interpretation. The `winget` command should be available on modern Windows, and would avoid having to install and buy into the Conda ecosystem as a _default recommendation_.